wa/workloads/gfxbench/__init__.py

```diff
@@ -0,0 +1,61 @@
+#    Copyright 2014-2016 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import re
+
+from wa import ApkUiautoWorkload, WorkloadError
+
+
+class Gfxbench(ApkUiautoWorkload):
+
+    name = 'gfxbench'
+    package_names = ['net.kishonti.gfxbench.gl.v50000.corporate']
+    regex_matches = [re.compile(r'Car Chase score \(([\d.]+)'),
+                     re.compile(r'Car Chase Offscreen score \(([\d.]+)'),
+                     re.compile(r'Manhattan 3.1 score \(([\d.]+)'),
+                     re.compile(r'1080p Manhattan 3.1 Offscreen score \(([\d.]+)'),
+                     re.compile(r'1440p Manhattan 3.1 Offscreen score \(([\d.]+)'),
+                     re.compile(r'Tessellation score \(([\d.]+)'),
+                     re.compile(r'Tessellation Offscreen score \(([\d.]+)')]
```

Comment:
Might this be similar to the antutu workload in that some tests are not supported on all devices resulting in no score being present? 
You allow for not being able to convert the expected score to a float below however your regexes will only match on numbers.

Reply:
We only get numbers as a result so is that a problem? I thought I had matched the antutu workload behaviour here?

Reply:
If you can only get number in that location that will be fine, although for the antutu workload on some older devices not all the tests were supported and therefore produced output like `'Tessellation score Not Supported` causing the workload to fail, therefore in https://github.com/ARM-software/workload-automation/pull/684 we change it to allowing matching of any output following the predefined statements and then handle any invalid conversions rather than only matching on numbers directly.